usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -2870,6 +2880,43 @@ static int get_curve_type_from_template(TEMPLATE *tmpl)
     return curve_type;
 }
 
+CK_BBOOL ep11_check_pkey_attrs(CK_ATTRIBUTE_PTR attrs,
```

Comment:
Naming: currently all the protected key related routines in ep11_specific.c are grouped together as ep11tok_pkey_xxxx. I would rename this routine to ep11tok_pkey_check_attrs. Also the routine below: ep11tok_pkey_aes_xts_check or ep11tok_pkey_check_aes_xts. And I would copy them into the block of the other ep11tok_pkey_xxx routines.